### PR TITLE
fix(VSelect): display placeholder text

### PR DESCRIPTION
**Pull request by guxuerui**
## Description

Missing feature in V3.0, `VSelect` component property: `placeholder`.
behaves as same as in v.2x.

## Motivation and Context

Missing feature

## How Has This Been Tested?

visually tested with `next` branch

## Markup:

<details>

```vue
<template>
  <v-app>
    <v-container>
      <v-select
        v-model="selected"
        :items="items"
        variant="outlined"
        placeholder="please select"
      />
    </v-container>
  </v-app>
</template>

<script>
  import { ref } from 'vue'
  export default {
    name: 'Playground',
    setup () {
      const selected = ref('')
      const items = ref([
        { title: 'item1', value: 1 },
        { title: 'item2', value: 2 },
      ])
      return {
        selected,
        items,
      }
    },
  }
</script>

```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VSelect/VSelect.tsx [hunk 1]
```diff
@@ -53,6 +53,8 @@ export const makeSelectProps = propsFactory({
     default: '$vuetify.noDataText',
   },
   openOnClear: Boolean,
+  placeholder: String,
+  persistentPlaceholder: Boolean,
```
Comment (KaelWD) [inline packages/vuetify/src/components/VSelect/VSelect.tsx:57]: If you remove these from here they'll just be passed through automatically instead.

File: packages/vuetify/src/components/VSelect/VSelect.tsx [hunk 2]
```diff
@@ -169,12 +171,15 @@ export const VSelect = genericComponent<new <
...
           validationValue={ props.modelValue }
+          placeholder={ props.placeholder }
+          persistentPlaceholder={ props.persistentPlaceholder }
           class={[
             'v-select',
             {
               'v-select--active-menu': menu.value,
               'v-select--chips': !!props.chips,
               [`v-select--${props.multiple ? 'multiple' : 'single'}`]: true,
+              'v-select--selected': !!props.modelValue,
```
Comment (KaelWD) [inline packages/vuetify/src/components/VSelect/VSelect.tsx:182]:
```suggestion
              'v-select--selected': mode.value.length,
```

**Comment by guxuerui** [comment]
Thank you very much for your guidance!  I have modified the code and resubmitted it~

File: packages/vuetify/src/components/VSelect/VSelect.sass [hunk 3]
```diff
@@ -16,9 +16,10 @@
     .v-field__input
       > input
```
[the remark this answers is not in the document]
Comment (guxuerui) [reply packages/vuetify/src/components/VSelect/VSelect.sass:17]: Tnank you very much! I have modified the code
```css
  &--selected
    .v-field
      .v-field__input
        > input
          opacity: 0
          transition: none
          pointer-events: none
```

File: packages/vuetify/src/components/VSelect/VSelect.tsx (hunk 1 above)
Comment (guxuerui) [reply packages/vuetify/src/components/VSelect/VSelect.tsx]: Yes, I removed them

File: packages/vuetify/src/components/VSelect/VSelect.tsx (hunk 2 above)
Comment (guxuerui) [reply packages/vuetify/src/components/VSelect/VSelect.tsx]: I tested with `model.value.length` and the result is always true.  But I tested with `props.modelValue.length` and it behaves normally, should I use this property?
```vue
   'v-select--selected': props.modelValue.length,
```

**Review by KaelWD — APPROVED** [review]

**Pull request merged**